scripts/cnv_wdl/somatic/cnv_somatic_panel_workflow.wdl

```diff
@@ -44,8 +45,10 @@ workflow CNVSomaticPanelWorkflow {
     if (!is_wgs) {
         call CNVTasks.PadTargets {
             input:
-                targets = targets,
-                gatk_jar = gatk_jar
+                # This is a bit of a hack.  The task will fail if targets is not defined when it gets here.
```

Comment:
I like it just as a warning. If you feel strongly, I'll get rid of it.

No action.